### PR TITLE
Add supported RPC methods

### PR DESCRIPTION
**Pull request by fabiorigam**
## Summary by CodeRabbit

- **Documentation**
	- Added a new section titled "JSON RPC Methods Support Status" in the README, providing a detailed table of various RPC methods and their support status.
	- Introduced a notes section to clarify the definitions of the support statuses for improved user understanding.

### CODE REVIEW

File: packages/rpc-proxy/README.md [hunk 1]
```diff
@@ -190,3 +190,56 @@ docker run -d -p 8545:8545 -v ./config.json:/app/config.json -t vechain-rpc-prox
 
 If you do not pass a config.json file, the default solo network standard configuration will be used. Make sure to
 provide your desired configuration file.
+
+## Supported methods
+
+The following RPC methods are supported via `sdk-rpc-proxy` in VeChain:
```
Comment (leszek-vechain) [inline packages/rpc-proxy/README.md:196]: should we also add some section to methods we did not implement yet and ones we wont implement ?

**Comment by claytonneal** [comment]
could we add a markdown table instead
we could classify the method as:

- fully supported
- partially supported (a column in the table can give the details)
- not supported

File: packages/rpc-proxy/README.md [hunk 2]
```diff
@@ -190,3 +190,56 @@ docker run -d -p 8545:8545 -v ./config.json:/app/config.json -t vechain-rpc-prox
...
+ - evm_mine
+ - net_listening
+ - net_peerCount
+ - net_version
+ - txpool_content
+ - txpool_contentFrom
+ - txpool_inspect
+ - txpool_status
+ - web3_clientVersion
+ - web3_sha3
```
Comment (claytonneal) [inline packages/rpc-proxy/README.md:245]: to me the idea of the documentation is to list:

- fully supported
- partially supported
- not supported

e.g. the uncle methods are partially supported - i.e. they exist but dont return anything

Comment (claytonneal) [reply packages/rpc-proxy/README.md:245]: yes something like that

**Review by leszek-vechain — APPROVED** [review]

**Review by victhorbi — APPROVED** [review]
The recommendations from coderabbitai look good, but I'm not sure if they capture everything. I prefer that we expand in the rpc-proxy documentation instead

**Pull request merged**